### PR TITLE
Make plan-preview comment more readable in case there are some unchanged applications

### LINKED ISSUE #2298: [plan-preview] Allow hiding no-diff application
**What would you like to be added**:

Currently, plan-preview is showing all applications that could be deployed after got merging. That is good for sure.
But sometimes an application got touched but not caused any diffs. (e.g. updating `.pipe.yaml` file or updating the tag of a remote source which refers to the same source code).
Some users want to hide those applications from the plan-preview results for a better look.

**Why is this needed**:

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #2298

**Does this PR introduce a user-facing change?**:

```release-note
Make plan-preview comment more readable in case there are some unchanged applications
```

### CODE REVIEW

File: dockers/actions-plan-preview/planpreview.go [hunk 1]
```diff
@@ -202,3 +223,18 @@ func makeCommentBody(event *githubEvent, r *PlanPreviewResult) string {
...
 	return b.String()
 }
+
+func groupApplicationResults(apps []ApplicationResult) (changes, pipelines, quicks []ApplicationResult) {
+	for _, app := range apps {
+		if !app.NoChange {
+			changes = append(changes, app)
+			continue
+		}
+		if app.SyncStrategy == "PIPELINE" {
```
Comment (nakabonne) [inline dockers/actions-plan-preview/planpreview.go:233]: nit
```suggestion
		if app.SyncStrategy == model.SyncStrategy_PIPELINE.String() {
```

Comment (nghialv) [reply dockers/actions-plan-preview/planpreview.go:233]: This actions package is stand alone so it will not use any PipeCD's packages.

Comment (nakabonne) [reply dockers/actions-plan-preview/planpreview.go:233]: True

**Comment by nakabonne** [comment]
deal
/lgtm

File: dockers/actions-plan-preview/Dockerfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ FROM golang:1.16.5-alpine3.14 as builder
 COPY . /app
 RUN cd /app && go build -o /plan-preview .
 
-FROM gcr.io/pipecd/pipectl:v0.13.0
+FROM gcr.io/pipecd/pipectl:v0.13.1-4-gec4b78b
```
Comment (khanhtc1202) [inline dockers/actions-plan-preview/Dockerfile:5]: I'm a bit worried here 👀  How about adding a TODO here or create an issue to remind us to update this Pipectl version of `v0.13.2` once it's rolled out?

Comment (nghialv) [reply dockers/actions-plan-preview/Dockerfile:5]: I'm on the train now so can you give me a suggestion to apply from my phone. 😀

Comment (khanhtc1202) [reply dockers/actions-plan-preview/Dockerfile:5]:
```suggestion
# TODO: Update pipectl version to v0.13.2 once it's rolled out.
FROM gcr.io/pipecd/pipectl:v0.13.1-4-gec4b78b
```
🙌

Comment (nghialv) [reply dockers/actions-plan-preview/Dockerfile:6]: Thank you. Applied it!

**Comment by khanhtc1202** [comment]
🚀 
/approve

File: dockers/actions-plan-preview/Dockerfile (hunk 2 above)
Comment (nakabonne) [reply dockers/actions-plan-preview/Dockerfile:6]: always fantastic to see those helping each other out.

**Pull request merged**